dali/pipeline/executor/executor.h

```diff
@@ -293,12 +293,18 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunCPU() {
           TimeRange::kCyan);
       op.Run(&ws);
     }
-  } catch (std::runtime_error &e) {
+  } catch (std::exception &e) {
     exec_error_ = true;
     QueuePolicy::SignalStop();
     std::lock_guard<std::mutex> errors_lock(errors_mutex_);
     errors_.push_back(e.what());
     // Let the ReleaseIdx chain wake the output cv
+  } catch (...) {
```

Comment:
instead of duplicating the code, let's have a function `handle_exception(const char* msg)` and reuse it in both catch blocks

Reply:
Done.